README.md

```diff
@@ -0,0 +1,21 @@
+# Cruise ships
```

Comment:
The comments in the review for your virtual pet apply here too I think. Its worth spending a bit of time to get the readme looking good as its the first thing anybody sees when they find you project

Heres a helpful cheatsheet [guide to markdown](https://www.markdownguide.org/cheat-sheet/) if you want some ideas